client/src/components/landingpage/AniCard.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 345,
+  },
+  media: {
+    height: 300,
+    width:"100%"
+  },
+  adjust: {
+      marginLeft:"auto",
+      marginRight:"auto"
+  }
+});
+
+export default function AniCard({number}) {
```

Comment:
Is the `number` prop required? It didn't have any use in the component, so we can safely remove it.